packages/core-components/src/components/DependencyGraph/Edge.tsx

```diff
@@ -126,7 +126,12 @@ export function Edge<EdgeData>({
   return (
     <>
       {path && (
-        <path data-testid={EDGE_TEST_ID} className={classes.path} d={path} />
+        <path
```

Comment:
While I really like this change I do think it should be opt-in given that it might be possible for some existing Adopters that this creates confusion as notes by @freben here: https://github.com/backstage/backstage/issues/21587#issuecomment-1833698235

Reply:
@awanlin yep, I saw that comment and it wasn't convincing, did you see my answer there - https://github.com/backstage/backstage/issues/21587#issuecomment-1889445662? It's a pity that the removal of the arrow heads (as part of https://github.com/backstage/backstage/pull/13673) wasn't made opt-in (or at least opt-out), I'd argue that it's the current behaviour that creates confusion, I tried to illustrate that with the screenshots in the PR description.

If the worry is about existing adopters, wouldn't it be acceptable to make it opt-out instead? I believe that changing the default behaviour will improve the understandability of any non-trivial diagram in Backstage, so I'd love to have a feeling at least that I managed to explain myself, even if at the end I won't convince you or @freben.

Reply:
Yes, I did see the various comments. We generally lean to opt-in for these changes and use the changeset to note how to enable them in the past. I'm also basing my thinking on the data I have - I'm very active on Discord helping people and the arrows are rarely a problem that has ben bought up from that group of people and the issue you logged doesn't have many comments or interactions. This leans my thinking again to making this opt-in.

To be clear I think this is an improvement and will be helpful. Also, I'm not the final say on this, that I defer to the Maintainers like @freben to decide on and will be fine with whatever they decide. 👍

Reply:
Thanks for the explanation and your time. I prefer simpler solutions since they are easier to maintain but I understand that flexibility might be more important for a project like Backstage, so it shall be configurable. I'd still advocate for making it at least opt-out since there are cases when the current rendering is misleading and I don't really see how it could harm the existing adopters. Even the videos in the catalog graph documentation contain arrow heads: https://github.com/backstage/backstage/blob/master/plugins/catalog-graph/README.md. Nevertheless even having it opt-in scratches my itch, so I'll push a new iteration soon.

I'd be happy to change to opt-out if there is interest of course :wink:.